webapp/channels/src/components/create_user_groups_modal/create_user_groups_modal.tsx

```diff
@@ -187,10 +184,7 @@ export default class CreateUserGroupsModal extends React.PureComponent<Props, St
                                         this.goBack();
                                     }}
                                 >
-                                    <LocalizedIcon
-                                        className='icon icon-arrow-left'
-                                        ariaLabel={{id: t('user_groups_modal.goBackLabel'), defaultMessage: 'Back'}}
-                                    />
+                                    <i className='icon icon-arrow-left' aria-label={t('user_groups_modal.goBackLabel')}></i>
                                 </button>
```

Comment:
hi @MeenuyD please follow the instructions to get an idea on how we can do this PR, you can also refer to the already done pr which is linked in the ticket https://github.com/mattermost/mattermost/issues/25110#issue-1959027536

Reply:
Ok